assets/app/scripts/directives/nav.js

```diff
@@ -58,10 +58,11 @@ angular.module('openshiftConsole')
 
           sortedProjects = $filter('orderByDisplayName')(projects);
           options = _.map(sortedProjects, function(item) {
+            // truncates @ 35 as this prevents words from clobbering the caret
             return $('<option>')
-                      .attr("value", item.metadata.name)
+                      .attr('value', $filter('truncate')(item.metadata.name, 35))
```

Comment:
I would assume so. I haven't thoroughly assessed all the new style updates yet.  Initially assumed we wanted the select box to stretch to fit longer names, but it appears to be consistent width always, so probably not a problem.